.github/workflows/trivy-containers.yaml

```diff
@@ -30,8 +30,17 @@ jobs:
         image: ${{ fromJson(needs.build-matrix.outputs.images) }}
     
     steps:
+      - name: Login to Docker Hub
```

Comment:
What are we pulling from docker hub? Why login there?

Reply:
Fixed, authenticating to public ECR using CNCF account.